spec/formatting.md

```diff
@@ -214,8 +214,6 @@ the following steps are taken:
    If _options_ is missing, the mapping will be empty.
    For each _option_:
    - Resolve the _identifier_ of the _option_.
-   - If the _option_'s _identifier_ already exists in the resolved mapping of _options_,
-     emit a Duplicate Option Name error.
```

Comment:
Why remove this? An implementation might detect this during parsing (which, it sounds like, you're doing when populating the data model). But it might not (if it naively parses the message and doesn't evaluate options until formatting time). Does this directive do any harm?

Reply:
Because we don't do that for any of the data model errors, and keeping this here would create a requirement for throwing data model errors during formatting, rather than before.

We should probably move the syntax & data model error definitions out of `formatting.md` into their own doc, and make it clearer that a message must be validated against all of them before formatting.